File: Selenium/Project/Activity2.java
package seleniumActivities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity2 {
	WebDriver driver;
	
	@BeforeClass
	public void setUp(){
		driver = new FirefoxDriver();
		driver.get("http://alchemy.hguy.co/crm");
	}
	
	@Test
	public void url() {
		WebElement headerUrl = driver.findElement(By.xpath("//img[@alt='SuiteCRM']"));
		String imageUrl = headerUrl.getAttribute("src");
		System.out.println("The url of the header image is: "+imageUrl);
	}
	
	@AfterClass
	public void tearDown() {
		driver.quit();
	}

}

File: Selenium/Project/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
    driver.get("http://alchemy.hguy.co/crm")

    headerUrl = driver.find_element(By.XPATH,"//img[@alt='SuiteCRM']")
    imageUrl = headerUrl.get_attribute("src")
    print(f"The url of the header image is: {imageUrl}")
    driver.quit()

